### ISSUE TITLE
Why do we ask DMP contributors for phone number

### ISSUE THREAD

**Issue by dsisu** [DMPRoadmap/roadmap#2941]
Please complete the following fields as applicable:

**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**
V3.0.2

**Expected behaviour:**
Ideally, we should only provide email address for contributors to a plan.

**Actual behaviour:**
When adding a contributor to a DMP, we ask for phone number too. Why is it needed? 

Please note that a phone number could be classed as more private information during Covid lockdown (people who work from home will have to provide mobile phone number or home telephone number rather than office number).

**Steps to reproduce:**
Add contributors to a plan.

**Comment by johnpinto1** [comment]
@dsisu If I understand this correctly, you want to have the phone number removed.
Is this agreed @mariapraetzellis & @briri for UC3 too?

**Comment by briri** [comment]
Yes, in fact we have hidden it on the DMPTool, so safe to remove.

**Comment by raycarrick-ed** [comment]
or make it configurable

**Comment by dsisu** [comment]
@johnpinto1 Sorry for the delay in responding. I don't mind what happens with the phone no, but I feel it shouldn't be a compulsory field.

**Comment by magdalenadrafiova** [comment]
works on dev @dsisu to close once deployed to live and retested!

**Comment by pherterich** [comment]
@raycarrick-ed - on a side note, can we check for DMPonline where this has been stored and make sure we delete the info we have collected so far? Not sure anyone used it, but just in case.

**Comment by raycarrick-ed** [comment]
Yes we can do that. Just want to double check I have understood correctly before deleting anything.
There are 9529 contributors with phone numbers at the moment. @pherterich you want these phone numbers deleted?
Just for clarity, we don't store phone numbers for users, just for the contributors when someone fills in that form.

**Comment by pherterich** [comment]
Maybe hold on for a moment and don't delete immediately - I need to think it through properly.
If we do not display that info (do we continue to display that, just don't allow people to enter new phone numbers?) and we don't make use of them, I just think that this is additional potentially personal information that we shouldn't have/store longer than needed. I didn't expect so many people filling that field though, so we probably should wait a bit in case someone was relying on that field. Probably something we should set ourselves a reminder for to look at in a few weeks though to decide what we do with it.

**Comment by pherterich** [comment]
Closing here - will open something for DMPonline only on the data stored so far.
